test/utils/wasm3_engine.cpp

```diff
@@ -41,8 +41,8 @@ namespace
 const void* env_adler32(IM3Runtime /*runtime*/, uint64_t* stack, void* mem)
 {
     uint64_t* ret = stack;
-    const uint32_t offset = *(uint32_t*)(stack++);
-    const uint32_t length = *(uint32_t*)(stack++);
+    const uint32_t offset = static_cast<uint32_t>(*stack++);
+    const uint32_t length = static_cast<uint32_t>(*stack);
```

Comment:
I used `stack[0]` and `stack[1]` for readability here.

Reply:
And to that effect we could use `stack[0] = fizzy::tests:adler32`, couldn't we?

Reply:
Done, thanks.